### ISSUE TITLE
Data is light grey and field names are darker and bigger, this could be inverted so values are easier to see.

### ISSUE THREAD

**Issue by edwardcodes** [FAC-GM/gm#10]
<img width="512" alt="field-names-should-be-lighter-values-darker" src="https://cloud.githubusercontent.com/assets/8533973/10643429/b39ae156-781a-11e5-895f-501be0764d02.png">

**Comment by nelsonic** [comment]
Agree. data should be easier to "*scan*". I would use light grey for the field names and dark blue for the values to make it clearer.

**Comment by heron2014** [comment]
@edwardcodes @nelsonic thanks for the suggestion!  I will work on some changes! :muscle:
